### ISSUE TITLE
Strange "banding" occurs in Luigi's Mansion on Ishiiruka Dolphin

### ISSUE THREAD

**Issue by HenrikoMagnifico** [Tinob/Ishiiruka#113]
I noticed that, while playing Luigi's Mansion on Ishiiruka Dolphin, strange "banding" of shadows appears on all moving objects on screen (Luigi, Toad, the Poltergust 3000, all the ghosts etc). I have tried disabling and enabling various settings but nothing seems to solve this issue. The problem does **not** occur on the latest Dolphin development build. It also does not occur whenever Luigi is in Professor E. Gadd's lab or when in the gallery.

[Here is a screenshot of what it looks like.](https://i.imgur.com/dGidzD8.png)

Using Ishiiruka-Dolphin 1050 64x

**Comment by Tinob** [comment]
can you share your confguration because the only way i get something like that is by enabling auto bumps and putting the threshold and intensity all the way up, otherwise it works perfecly for me.

**Comment by HenrikoMagnifico** [comment]
@Tinob Where is the config file located? I only find a bunch of game settings in \Ishiiruka Dolphin\Sys\GameSettings

**Comment by Tinob** [comment]
if you are on windows is in Documents/Dolphin Emulator/Config

**Comment by HenrikoMagnifico** [comment]
@Tinob I'm a bit unsure which of the files you need so here is the entire Config folder :) https://drive.google.com/file/d/1zsrksLaSPti-Hquy1H3_RDIS9SoYLSb9/view?usp=sharing

**Comment by Tinob** [comment]
Im unable to reproduce the issue, does it happends with all the backends?

**Comment by HenrikoMagnifico** [comment]
@Tinob Yup, it does. It appears to be some kind of issue with shaders in Luigi's Mansion (even though I'm not using them)

**Comment by Tinob** [comment]
did you tested older version have an idea on when the issue started?

**Comment by HenrikoMagnifico** [comment]
@Tinob Sorry that I couldn't respond immediately, I have been away from town for the weekend. 

I can't really recall when this issue started occurring, however I am doing some testing right now.

**Comment by Tinob** [comment]
wow that is really weird stable version 776 and 1050 share the exact graphic code so graphic output should be exactly the same, can you test with older version of the non stable branch?

**Comment by HenrikoMagnifico** [comment]
@Tinob Okay I have done some further testing and come to the conclusion that Ishiiruka-Dolphin 1028 was the last version that did not have this bug present. All versions after this has the bug.
